lib/gettext.ex

```diff
@@ -0,0 +1,5 @@
+if Mix.env == :test do
```

Comment:
This is an unfortunate side effect but we need some module in the config so that the tests can use it to translate the errors.